api/user.go

```diff
@@ -31,9 +32,26 @@ type UserAPI struct {
 	userID        int
 }
 
+type userReq struct {
+	UserName string `json:"username"`
```

Comment:
I just follow the project.go under /api. It's ok to use the model.User, but I am not sure add the extension, like `json:"password"`, will impact the DB schema.